Automattic/jetpack
# Codecoverage: fix reports

#### Changes proposed in this Pull Request:

* ~Disables a code coverage for `codesniffer` package. #17729 should fix the underlying problem, so might not be needed.~
* Refactors how PHP code coverage tests are being run
* Run code coverage for multisite tests
* Run code coverage for legacy sync tests 

#### Jetpack product discussion


n/a

#### Does this pull request change what data or activity we track or use?


n/a

#### Testing instructions:






* Check code coverage job in Travis. Make sure it is green, and all the tests are being run
*

#### Proposed changelog entry for your changes:

* n/a
